temba_client/clients.py

```diff
@@ -370,6 +372,9 @@ def _request_wth_rate_limit_retry(self, method, url, params=None, body=None):
                 retries += 1
 
                 if retries < MAX_RETRIES and ex.retry_after:
-                    time.sleep(ex.retry_after)
+                    server_wait_time = ex.retry_after
```

Comment:
Whilst you're at it, could you log to the console how long you're backing off?

Reply:
Was going to do that but the rest of the library doesn't have any logging/printing configured...